### ISSUE TITLE
Vimeo Player Accessibility

### ISSUE THREAD

**Issue by brndnb** [sanderheilbron/vimeo.ga.js#21]
Is there a way to make the video image accessible?

On the demo page I can't tab through to select the image, so I can imagine this would also be a problem on screen readers. With the current approach, all of the easily accessible Vimeo buttons are made inaccessible with this plugin until clicking the video image.

Is there no way to make this work nicely with screen readers? Or am I missing something?

**Comment by sanderheilbron** [comment]
The placeholder image is not supported by the script. 

I've added it to the demo page to improve the perceived performance and reduce the total load time of the page.

**Comment by brndnb** [comment]
Excellent! Thanks for the clarification.
